incubator/group/msg.go

```diff
@@ -231,18 +224,18 @@ func (m *MsgProposeBase) ValidateBasic() error {
 	return nil
 }
 
-var _ sdk.Msg = &MsgCreateGroupAccountStd{}
+var _ sdk.Msg = &MsgCreateGroupAccount{}
 
-func (m MsgCreateGroupAccountStd) Route() string { return ModuleName }
-func (m MsgCreateGroupAccountStd) Type() string  { return msgTypeCreateGroupAccountStd }
+func (m MsgCreateGroupAccount) Route() string { return ModuleName }
+func (m MsgCreateGroupAccount) Type() string  { return msgTypeCreateGroupAccount }
 
 // GetSigners returns the addresses that must sign over msg.GetSignBytes()
-func (m MsgCreateGroupAccountStd) GetSigners() []sdk.AccAddress {
-	return []sdk.AccAddress{m.Base.Admin}
+func (m MsgCreateGroupAccount) GetSigners() []sdk.AccAddress {
+	return []sdk.AccAddress{m.Admin}
 }
 
 // GetSignBytes returns the bytes for the message signer to sign on
-func (m MsgCreateGroupAccountStd) GetSignBytes() []byte {
+func (m MsgCreateGroupAccount) GetSignBytes() []byte {
 	var buf bytes.Buffer
 	enc := jsonpb.Marshaler{}
 	if err := enc.Marshal(&buf, &m); err != nil {
```

Comment:
Maybe we should even just panic here because amino isn't supported? Not sure

Reply:
should we just have 
```go
func (m MsgCreateGroupAccount) GetSignBytes() []byte {
        bz := ModuleCdc.MustMarshalJSON(m)
	return sdk.MustSortJSON(bz)
}
```
 instead (as in cosmos-sdk x/gov)?

Reply:
Yes, if we are going to support amino then we should take that approach.